packages/core/package.json

```diff
@@ -65,13 +65,19 @@
     "webpack": "5.90.0"
   },
   "peerDependencies": {
-    "ember-source": "^4.0.0"
+    "ember-source": "^4.0.0",
+    "ember-data": ">=4.0.0"
```

Comment:
Potentially even `*` as ember-caluma doesn't use it and doesn't care about the version of ember-data at all.

Reply:
sure, fixed